### PR TITLE
Fixes #60: Replaces `<paper-ripple>` use with `PaperRippleBehavior`.

### LINKED ISSUE #60: paper-tab is broken on safari browsers
It throws an error "TypeError: null is not an object (evaluating 'this.parentNode.nodeType')" in Safari 7.1.8 (9537.85.17.9.1) (Mac Desktop), and the same error happens in iOS 8.4 Safari. This error breaks other polymer events (like attached) in other components on the same page, making this component unusable on mentioned browsers.

I tracked down the problem and discovered that it is related to paper-ripple element. This element is inside a ```<template is="dom-if">``` tag, and it apparently receives a null pointer when trying to find its parentNode. (Obs: this occurs only in those browser, this problem never occurred on Chrome or Firefox)

I checked paper-button source code to see how they did solve this problem (that element has a noink property too and it works fine over there). I found that they hide the paper-ripple using a CSS selector instead of a dom-if tag.

If we want to use the same solution in paper-tabs, we can change ```paper-tab.html``` by removing ```<template is="dom-if" ...></template>``` tags and adding an attribute ```noink=[[noink]]``` to the paper-ripple. And finally, we can reflect noink atribute to the host. This way we can control paper-ripple display using CSS. Here are the changes:

```diff
@@ -97,6 +97,10 @@
       -webkit-flex-basis: 0.000000001px;
       flex-basis: 0.000000001px;
     }
+
+    :host[noink] > paper-ripple {
+      display: none;
+    }
   </style>
 
   <template>
@@ -105,9 +109,7 @@
       <content></content>
     </div>
 
-    <template is="dom-if" if="[[!noink]]">
-      <paper-ripple id="ink" initial-opacity="0.95" opacity-decay-velocity="0.98"></paper-ripple>
-    </template>
+    <paper-ripple id="ink" noink="[[noink]]" initial-opacity="0.95" opacity-decay-velocity="0.98"></paper-ripple>
 
   </template>
 
@@ -133,7 +135,8 @@
        */
       noink: {
         type: Boolean,
-        value: false
+        value: false,
+        reflectToAttribute: true
       }
 
     },
```

PS: I checked the paper-ripple object and it suppresses pointer-events when it has a noink attribute correctly configured.

### PR DESCRIPTION
**Pull request by bicknellr**
:pig: `noink`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      fi
+before_script:
+- npm install web-component-tester
+- npm install bower
+- export PATH=$PWD/node_modules/.bin:$PATH
+- bower install
+env:
+  global:
+  - secure: OlScKe6WdchCYNKA8ZLT7W9l/OxGqQ221k+pJigmxOpE8nEQ6ScY8EmZZk6RyQDeFrCnCbfc1/TyprucZ1K53e1D3ljivSqDWz9R22jygA/PtEpd+aI0LnYtX/PHT78TjYtt32KP0scGNf+GYity7MxffNaf7n15ICQHGvA1RTs=
+  - secure: acJCTBwJDynNAyxsPy5/5OgdBuCrmxfUGs0Vifa7lrCFDKfbDfoQ4fI84rkIrfaK1MSUR5OrmWfFuZEmDG93jbStnkevnODc07BSc8zbWfFyOqfJvelQIo6Vtd8xL8IwsB8hD7UO4d6sWkbuLAokVaX9I4ZyEUUGhqwEk1j3Hcg=
```
Comment (cdata) [inline .travis.yml]: :100:

File: paper-tab.html [hunk 2]
```diff
@@ -83,9 +83,8 @@
       font-weight: 700;
     }
 
-    #ink {
+    paper-ripple {
```
Comment (cdata) [inline paper-tab.html]: :+1:

File: paper-tab.html [hunk 3]
```diff
@@ -122,23 +117,10 @@
 
     behaviors: [
       Polymer.IronControlState,
-      Polymer.IronButtonState
+      Polymer.IronButtonState,
+      Polymer.PaperRippleBehavior
```
Comment (cdata) [inline paper-tab.html]: :sparkles:

File: paper-tab.html [hunk 4]
```diff
@@ -147,6 +129,12 @@
       down: '_updateNoink'
     },
 
+    ready: function() {
+      var ripple = this.getRipple();
+      ripple.initialOpacity = 0.95;
+      ripple.opacityDecayVelocity = 0.98;
```
Comment (cdata) [inline paper-tab.html]: :point_up: :point_up_2: :+1:

File: paper-tabs.html [hunk 5]
```diff
@@ -319,6 +320,19 @@
       this.setScrollDirection('y', this.$.tabsContainer);
     },
 
+    _noinkChanged: function(noink) {
+      var childTabs = Polymer.dom(this).querySelectorAll('paper-tab');
+      childTabs.forEach(noink ? this._setNoinkAttribute : this._removeNoinkAttribute);
```
Comment (cdata) [inline paper-tabs.html]: :mortar_board:

**Comment by cdata** [comment]
It looks like there may be a good opportunity to add a test here for the behavior of `noink` on `paper-tabs`. A test could assert that all `paper-tab` elements get the value of `noink` set correctly on them as the `paperTabs.noink` changes. WDYT?

**Comment by bicknellr** [comment]
That's a good idea; I'll add a note about this behavior in the docs too.

**Comment by bicknellr** [comment]
@cdata, tests / documentation for `noink` attribute propagation are now in.

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**